src/click/termui.py

```diff
@@ -227,6 +225,10 @@ def confirm(
         "y/n" if default is None else ("Y/n" if default else "y/N"),
     )
 
+    # Initialise rv to avoid a
+    # `variable might be referenced before assignment` warning
```

Comment:
This doesn't happen.

Reply:
Oh?, I added that because I was seeing a warning in Pycharm